test/standalone/Soil/climate_drivers.jl

```diff
@@ -119,6 +119,11 @@ for FT in (Float32, Float64)
             )
 
             Y, p, coords = initialize(model)
+            @test propertynames(p.soil.sfc_conditions) ==
+                  (:lhf, :shf, :vapor_flux, :r_ae)
+            @test :top_bc ∈ propertynames(p.soil)
```

Comment:
could we instead test
```
@test propertynames(p.soil.sfc_conditions) ==
       (:sfc_conditions, :top_bc, :bottom_bc, :R_n)
```
?
Maybe this would be a stricter test bc it would also check that we don't add anything extra

Reply:
updated this!